prover/drivers/base.md

```diff
@@ -38,6 +43,8 @@ module PROVER-CONFIGURATION
         <declarations>
           <declaration multiplicity="*" type="Set">  .K </declaration>
         </declarations>
+        <hooked-smt-sorts> .Map </hooked-smt-sorts>
```

Comment:
Shouldn't these just be axioms? Why do we need a new cell?

Reply:
Theoretically, we do not need a new cell and can keep `hook-smt-sort` and `hook-smt-symbol` in the `<declarations>` cell as a declaration, but it slows down the prover significantly (smoke-tests take 15 minutes instead of 3).

Reply:
I wonder if this is an issue with the ocaml backend. Do the smoke tests really finish in 3 minutes on your machine?! It takes more than 30 on mine.

Reply:
It probably was an issue with the ocaml backend; under llvm backend, the times are rougly equal. (something over 1.5 minutes on 16-core machine (more precisely, 8-core + hyperthreading)).